hypha/apply/funds/templates/funds/includes/status-block.html

```diff
@@ -3,11 +3,10 @@
     <section class="section">
         <ul class="status-block">
             {% for status, data in status_counts.items %}
-                <li class="status-block__item">
-                    <h5 class="status-block__title">{{ data.name }}</h5>
-                    <p class="status-block__info">{{ data.count }} {{ type }}</p>
-                    <a class="status-block__link" href="{{ data.url }}">{% trans "View" %}</a>
-                </li>
+                <a class="status-block__item {% if data.is_active %}status-block__item--active{% endif %}" href="{{ data.url }}">
+                    <h5 class="status-block__title text-black">{{ data.name }}</h5>
```

Comment:
Overusing heading tags is a left over from the old Hypha design. I think we should take the opportunity here to update it. I think just replacing it with "strong" tag would be god, making it a bit smaller. The p-tag below could get the standard body font size. I think that would look nicer and take up less space.

@abhiXII What do you think?